zetaclient/chains/evm/signer/outbound_data.go

```diff
@@ -194,9 +198,9 @@ func getDestination(cctx *types.CrossChainTx, logger zerolog.Logger) (ethcommon.
 		chainID := big.NewInt(cctx.InboundParams.SenderChainId)
 
 		logger.Info().
-			Str("cctx.index", cctx.Index).
-			Int64("cctx.chain_id", chainID.Int64()).
-			Msgf("Abort: reverting inbound")
+			Str("cctx_index", cctx.Index).
+			Int64("cctx_chain_id", chainID.Int64()).
```

Comment:
```suggestion
			Int64("chain", chainID.Int64()).
```
Let's use `chain` for all chain ID I think, same comment for other location